rest/ProductService.js

```diff
@@ -5,8 +5,8 @@ import IllegalArgumentError from "./IllegalArgumentError.js";
 const EMPTY_ARGUMENT_ERR = "Body contains an empty argument!";
 const INVALID_EXPIRY_DATE_ERR = "Expiry date is in the past!";
 const INVALID_QUANTITY_ERR = "Quantity cannot be less than 0!";
-const INVALID_IS_ACTIVE_ERR = "Product cannot be deleted if active";
-const INVALID_ASSOCIATED_ORDER = "Cannot delete product with associated orders";
+const INVALID_DELETE_ACTIVE_ERR = "Product cannot be deleted if active";
```

Comment:
DELETE_ACITVE_ERR = "Product cannot be deleted while active"

what is INVALID_DELETE_ACTIVE_ERR even supposed to mean